### PR TITLE
Make swapToFrame method faster

### PR DESCRIPTION
**Pull request by annehaley**
This PR makes the frame slider respond faster to change.

### CODE REVIEW

File: web_client/src/store/index.ts [hunk 1]
```diff
@@ -810,7 +812,9 @@ const {
           );
           frameData = result.frameData;
         }
+        const start = Date.now();
         sourceProxy.setInputData(frameData);
+        console.log('diff after set', Date.now() - start);
```
Comment (zachmullen) [inline web_client/src/store/index.ts]:
```suggestion
```

**Review by zachmullen — CHANGES_REQUESTED** [review]

File: web_client/src/store/index.ts (hunk 1 above)
Comment (annehaley) [reply web_client/src/store/index.ts]: Thanks!

**Review by zachmullen — APPROVED** [review]

File: web_client/src/store/index.ts [hunk 2]
```diff
@@ -49,9 +50,9 @@ function prepareProxyManager(proxyManager) {
       view.setOrientationAxesVisibility(false);
       view.getRepresentations().forEach((representation) => {
         representation.setInterpolationType(InterpolationType.NEAREST);
-        representation.onModified(() => {
+        representation.onModified(macro.debounce(() => {
           view.render(true);
-        });
+        }, 0));
```
Comment (zachmullen) [inline web_client/src/store/index.ts:55]: I think having a debounce with a 0 time parameter merits a comment explaining it.

Comment (annehaley) [reply web_client/src/store/index.ts:55]: Does the comment in [b5444a7](https://github.com/OpenImaging/miqa/pull/633/commits/b5444a7cce695b048450122c2fde71fe8862021f) suffice? Or should I explain further about the simultaneous changes?

Comment (zachmullen) [reply web_client/src/store/index.ts:55]: LGTM

**Pull request merged**